### ISSUE TITLE
Calling Save() doesn't flush to the stream

### ISSUE THREAD

**Issue by encrypt0r** [OfficeDev/Open-XML-SDK#294]
**Description**

I am writing a wrapper class for SpreadsheetDocument. **I would like to be able to Save the document more than once**. Currently the only way to force SpreadsheetDocument to flush to the stream is by calling Close() on it. Is there anyway to avoid that?

**Information**

- .NET Target: .Net Standard 2.0
- DocumentFormat.OpenXml Version: 2.7.2

**Repro**

```csharp
using DocumentFormat.OpenXml;
using DocumentFormat.OpenXml.Packaging;
using DocumentFormat.OpenXml.Spreadsheet;
using System;
using System.IO;

namespace OpenXmlDemo
{
    class Program
    {
        static void Main(string[] args)
        {
            using (var stream = new MemoryStream())
            {
                var document = SpreadsheetDocument.Create(stream: stream,
                                                          type: SpreadsheetDocumentType.Workbook,
                                                          autoSave: true);

                var workbookPart = document.AddWorkbookPart();
                var workbook = workbookPart.Workbook = new Workbook();
                var sheetsElement = new Sheets();
                workbook.AppendChild(sheetsElement);

                InsertSheet(document, workbookPart, sheetsElement, "Sheet A", 1);

                // Save the workbook and document
                workbook.Save();
                document.Save();

                // document.Close();

                // Returns an empty array if the document isn't closed
                // stream.Position = 0; // Some places suggested this, unfortunately this doesn't seem to help.
                var bytes = stream.ToArray(); 
                File.WriteAllBytes(@"D:\sheet.xlsx", bytes);
            }
        }

        private static Sheet InsertSheet(SpreadsheetDocument doc,
                                      WorkbookPart workbookpart,
                                      Sheets sheetsElement,
                                      string name,
                                      uint id)
        {
            WorksheetPart newWorksheetPart = workbookpart.AddNewPart<WorksheetPart>();
            newWorksheetPart.Worksheet = new Worksheet(new SheetData());

            string relationshipId = workbookpart.GetIdOfPart(newWorksheetPart);

            // Give the new worksheet a name.
            if (string.IsNullOrEmpty(name))
                name = "Sheet" + id;

            // Append the new worksheet and associate it with the workbook.
            Sheet sheet = new Sheet() { Id = relationshipId, SheetId = id, Name = name };
            sheetsElement.AppendChild(sheet);

            return sheet;
        }
    }
}
```

**Observed**

The stream is empty.

**Expected**

The stream should contain the document.

**Comment by twsouthwick** [comment]
I've encountered this as well. I think it may be a limitation of System.IO.Packaging, but I'll look into it

**Comment by twsouthwick** [comment]
This is blocked by a bug in .NET Core. I've opened an issue here: https://github.com/dotnet/corefx/issues/25273

**Comment by twsouthwick** [comment]
When this is fixed, the tests in #383 should be turned on

**Comment by twsouthwick** [comment]
@encrypt0r I have a PR open to enable this with a flag on `OpenXmlPackage.CanSave` indicating whether the platform fully supports saving. It will still only work on .NET Framework, but the flag should help understand at runtime if saving will be successful or not. Please check it out at #468 and leave any feedback. This would be released in a 2.9.0 release.
